Bugzilla/WebService/User.pm

```diff
@@ -298,14 +298,15 @@ sub get {
   foreach my $user (@$in_group) {
     my $user_info = filter $params,
       {
-      id             => $self->type('int',     $user->id),
-      real_name      => $self->type('string',  $user->name),
-      nick           => $self->type('string',  $user->nick),
-      name           => $self->type('email',   $user->login),
-      email          => $self->type('email',   $user->email),
-      can_login      => $self->type('boolean', $user->is_enabled ? 1 : 0),
-      iam_username   => $self->type('string',  $user->iam_username),
-      last_seen_date => $self->type('dateTime',  $user->last_seen_date),
+      id                 => $self->type('int',      $user->id),
+      real_name          => $self->type('string',   $user->name),
+      nick               => $self->type('string',   $user->nick),
+      name               => $self->type('email',    $user->login),
+      email              => $self->type('email',    $user->email),
+      can_login          => $self->type('boolean',  $user->is_enabled ? 1 : 0),
+      iam_username       => $self->type('string',   $user->iam_username),
+      last_seen_date     => $self->type('dateTime', $user->last_seen_date),
+      last_activity_date => $self->type('dateTime', $user->last_activity_date),
```

Comment:
`last_activity_time => $self->type('dateTime', $user->last_activity_time),`

Reply:
Done in d7c0a9a80d3ff73b460c2ce9c944778fbd8a3887